VaticanBAV/et/BAVet123.xml

```diff
@@ -0,0 +1,233 @@
+                        <msItem xml:id="ms_i2">
+                            <locus target="#67r"/>
+                            <title type="incomplete" ref="LIT7178SalotaMa"/>
+                            <textLang mainLang="gez"/>
+                            <incipit>
+                                ጸሎተ፡ ማዕድ፡ አቡነ፡ ዘበሰማያት፡ በል፡ ወአንብሮ፡ በቅድሜከ፡ ዘትጥዕሞ፡ ቅዱስ፡ ቅዱስ፡ ቅዱስ፡ እግዚአብሔር፡ ዘአውረድከ፡ መና፡ ለእስራኤል፡ በገዳም፡ 
+                                ወነአምን፡ ከመ፡ ኵሎ፡ ትክል፡
+                            </incipit>
+                            <explicit>
+                                ባርኮ፡ በአፈ፡ ኵሉ፡ ፍጥረት፡ ለአለመ፡ ዓለም። ኪርያላ<supplied reason="omitted">ይ</supplied>ሶን፡ በ፡ ፫ ጊዜ፡
+                                <choice><sic>ቦርክ፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ባርክ፡</corr></choice> ፫ ጊዜ፡ ወጠዓም፡ በፈሪሃ፡ እግዚአብሔር፡ ወዓይንከ።
+                                <gap reason="omitted" resp="PRS4805Grebaut PRS9530Tisseran"/>
+                            </explicit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i3">
+                            <locus from="68r" to="72v"/>
+                            <title type="complete" ref="LIT4099MashafaT">ትርጓሜ፡ ፊደል፡</title>
+                            <textLang mainLang="gez" otherLangs="am"/>
+                            <incipit>
+                                በስመ<supplied reason="omitted">፡</supplied> አብ፡ <gap reason="omitted" resp="PRS4805Grebaut PRS9530Tisseran"/> ትጓሜ<supplied reason="omitted">፡</supplied>
+                                <surplus>ትርጓሜ፡</surplus> ፊደል። ሀ፡ ብሂል፡ አብ፡ ሁ<supplied reason="omitted">፡</supplied> ብሂል፡ ወልድ፡ ሂ፡ ብሂል፡ መንፈስ፡ ቅዱስ፡ ሃ፡ ብሂል<supplied reason="omitted">፡</supplied>
+                                ቤተ፡ ክርስቲያን፡ ሄ<supplied reason="omitted">፡</supplied> ብሂል፡ ጥምቀት፡ ህ፡ ብሂል፡ ምሳሃ<supplied reason="omitted">፡</supplied> ደብረ፡ ጽዮን፡
+                                ሆ፡ ብሂል፡ ትንሣዔ፡ ሙታን፡
+                             </incipit>
+                            <explicit>
+                                <choice><sic>ተሰብኦ፡</sic><corr resp="PRS4805Grebaut PRS9530Tisseran">ተሰብአ፡</corr></choice> ወተሠገወ፡ እመንፈስ፡ ቅዱስ፡ በለበሰው፡ ሥጋ፡
+                                መንፈስ፡ ቅዱስን፡ ተቀበለ፡ በአምላክነቱ፡ ከአብ፡ ከመንፈስ፡ ቅዱስ፡ አክብሮ፡ መንፈስ፡ ቅዱስን፡ በለበሰው፡ ሥጋ፡ ሰጠው፡ በሰውነቱ፡ <gap reason="omitted" resp="PRS4805Grebaut PRS9530Tisseran"/>
+                            </explicit>
+                        </msItem>
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc> 
+                                <support>
+                                    <material key="parchment">                                
+                            </material>
+                            </support>
+                                <extent>
+                                    <measure unit="leaf">76</measure>
+                                    <measure unit="leaf" type="blank">24
```

Comment:
I think that measure should be closed before adding the locus elements of the blank leaves 
```
<measure unit="leaf" type="blank">24</measure>
                                        <locus from="1r" to="2v"/>
                                        <locus target="#67v"/>
                                        <locus from="72r" to="92v"/>
```
The guidelines are a bit unclear, I will update them.